Client/package.json

```diff
@@ -28,6 +28,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
     "bootstrap": "^4.3.1",
     "jquery": "^3.4.1"
+    "node-sass": "^4.5.3",
```

Comment:
why this is required?

Reply:
Agreed, also this is not required as such hence removing